cellxgene_schema_cli/cellxgene_schema/validate.py

```diff
@@ -29,6 +29,8 @@
 
 VISIUM_AND_IS_SINGLE_TRUE_MATRIX_SIZE = 4992
 VISIUM_11MM_AND_IS_SINGLE_TRUE_MATRIX_SIZE = 14336
+VISIUM_TISSUE_POSITION_MAX = (77, 127)  # (row,col)
+VISIUM_11MM_TISSUE_POSITION_MAX = (127, 223)  # (row,col)
```

Comment:
i would slightly prefer storing these as 4 separate variables, that way the reader isn't expected to understand the ordering of row/col in the tuple. but not a strong preference, so this isn't blocking

Reply:
fair enough.